config/locales/ja.yml

```diff
@@ -9,6 +9,7 @@ ja:
   text_description_of_search_box: 検索ボックスにキーワードを入力することで、選択肢を絞り込むことが出来ます。
   text_for_your_reference: "もし書き方がわからない場合は、「%{value}」が参考になるはずです。"
   text_disabled_customize: このプラグインによるメッセージのカスタマイズは無効になっています。
+  text_unable_to_customize_plugin_messages: "プラグインのメッセージはカスタマイズできません。"
```

Comment:
These messages (`:text_disabled_customize`, `:text_unable_to_customize_plugin_messages`) look the same because they use 'message', 'plugin', and 'customize'.

Reply:
```
このプラグインによるメッセージのカスタマイズは無効になっています。
↓
メッセージのカスタマイズはすべて無効になっています。
```
```
Message customization by this plugin is disabled.
↓
All message customizations are disabled.
```

上のように変更しようと思うのですが、どうでしょうか？
このプラグインによる〜のところが紛らわしいように見えました。
それを除けば「無効」と「カスタマイズできない」で意図が伝わるのではないかと思っています。

Reply:
:text_unable_to_customize_plugin_messages の
`プラグインのメッセージは` が redmine_message_customize 自身のことを示しているように見えて気になりました。

Reply:
確かにそんな感じがありますね。
https://github.com/ishikawa999/redmine_message_customize/pull/34#discussion_r322509014に加えて下のような変更でどうでしょうか・

```
プラグインのメッセージはカスタマイズできません。
↓
Redmine自体のメッセージのカスタマイズのみが可能です。Redmineに入っているプラグインのメッセージのカスタマイズは出来ません。
```

```
Impossible to customize the any plugin messages.
↓
You can only customize Redmine's own messages. The message of the plugins included in Redmine cannot be customized.
```

Reply:
https://github.com/ishikawa999/redmine_message_customize/pull/34#discussion_r322509014 の変更はOKと思います。

Reply:
> 確かにそんな感じがありますね。
> [#34 (comment)](https://github.com/ishikawa999/redmine_message_customize/pull/34#discussion_r322509014) に加えて下のような変更でどうでしょうか？
> 
> ```
> プラグインのメッセージはカスタマイズできません。
> ↓
> Redmine自体のメッセージのカスタマイズのみが可能です。Redmineに入っているプラグインのメッセージのカスタマイズは出来ません。
> ```
> 
> ```
> Impossible to customize the any plugin messages.
> ↓
> You can only customize Redmine's own messages. The message of the plugins included in Redmine cannot be customized.
> ```

注意書きなのに、長くなってしまいますね。。。

Reply:
確かに、2文にもなると長い気がします...

Redmineに入っているプラグインのメッセージのカスタマイズは出来ません。
The message of the plugins included in Redmine cannot be customized.

はどうでしょうか？
冗長な気がしますが、Redmineに入っているプラグインとすることで"このプラグイン"と限定してみられないようにすることを意図しています。

Reply:
良いと思います。（たしかに冗長な気はするけど）

Reply:
とりあえずこれで修正してみます。

Reply:
英文の方は主語が長すぎる気がしたので下のように変えようと思います
`Impossible to customize the messages of the plugins.`

pluginsを複数形にすることで誤解を避けようという意図です。

Reply:
良いと思います。

Reply:
修正しました. 21b0800

Reply:
確認しました。OKです。